### PR TITLE
Allow specifying cluster_ca as unencoded PEM

### LINKED ISSUE #39: Allow specifying unencoded cluster CA certficate
When running Concourse configured with the Kubernetes secret credential source, I can use the secret that gets dynamically generated for each cluster `ServiceAccount` to authenticate this resource to the cluster. The dynamic secret conveniently provides the token and CA certificate required by the resource and is easy to access with Concourse. For example, I should be able to do:

```yaml
resources:
  - name: helm-release
    type: helm
    source:
      namespace: ((concourse-deployer-token.namespace))
      cluster_url: "https://kubernetes.default.svc"
      cluster_ca: ((concourse-deployer-token."ca.crt"))
      token: ((concourse-deployer-token.token))
```

However this currently isn't possible because the resource expects a Base64-encoded certificate, and `((concourse-deployer-token."ca.crt"))` evaluates to the unencoded PEM.

Maybe we could have a new parameter like `cluster_ca_pem` that accepts the unencoded PEM? I'm happy to raise a PR to implement this if it seems suitable.

### PR DESCRIPTION
**Pull request by vixus0**
Try decoding value as Base64 and fallback to treating value as the
unencoded PEM.

Closes #39

### CODE REVIEW

File: assets/common.sh [hunk 1]
```diff
@@ -35,7 +35,8 @@ setup_kubernetes() {
         kubectl config set-cluster default --server=$cluster_url --insecure-skip-tls-verify=true
       else
         ca_path="/root/.kube/ca.pem"
-        echo "$cluster_ca" | base64 -d > $ca_path
+        (echo "$cluster_ca" | base64 -d || echo "$cluster_ca") > $ca_path
+        fi
```
Comment (Typositoire) [inline assets/common.sh:39]: I think you didn't mean to put a `fi` there ?

**Review by Typositoire — CHANGES_REQUESTED** [review]

File: assets/common.sh (hunk 1 above)
Comment (vixus0) [reply assets/common.sh]: Ha oops, fixed :)

**Comment by Typositoire** [comment]
This will be merged and available as latest within 10-20 minutes (Pray the docker hub gods!)

Then it'll be tagged as 1.13.0 soon after. ETA 30 minutes to be in Docker Hub

**Comment by vixus0** [comment]
@Typositoire thank you for the speedy turnaround! :)

**Pull request merged**